examples/integrations/cdi/pokemons/src/main/resources/META-INF/init_script.sql

```diff
@@ -17,9 +17,9 @@ INSERT INTO POKEMONTYPE VALUES (16, 'Dragon');
 INSERT INTO POKEMONTYPE VALUES (17, 'Dark');
 INSERT INTO POKEMONTYPE VALUES (18, 'Fairy');
 
-INSERT INTO POKEMON VALUES (1, 'Bulbasaur', 12);
-INSERT INTO POKEMON VALUES (2, 'Charmander', 10);
-INSERT INTO POKEMON VALUES (3, 'Squirtle', 11);
-INSERT INTO POKEMON VALUES (4, 'Caterpie', 7);
-INSERT INTO POKEMON VALUES (5, 'Weedle', 7);
-INSERT INTO POKEMON VALUES (6, 'Pidgey', 3);
+INSERT INTO POKEMON VALUES (1, 12, 'Bulbasaur');
+INSERT INTO POKEMON VALUES (2, 10, 'Charmander');
+INSERT INTO POKEMON VALUES (3, 11, 'Squirtle');
+INSERT INTO POKEMON VALUES (4, 7, 'Caterpie');
+INSERT INTO POKEMON VALUES (5, 7, 'Weedle');
+INSERT INTO POKEMON VALUES (6, 3, 'Pidgey');
```

Comment:
This is the root of the problem, right? That for some reason Hibernate 6.2.x has changed the order of DDL/column generation? I would like to better understand why that is.

Reply:
See #7433